python_modules/dagster/dagster/_core/definitions/metadata/source_code.py

```diff
@@ -1,5 +1,7 @@
+import abc
```

Comment:
Nitpick: we normally do `from abc import ABC, abstractmethod`. Worth being consistent.